docs/prover/approx/grounding.md

```diff
@@ -0,0 +1,285 @@
+(grounding)=
+# Quantifier Grounding 
+
+{term}`Quantified expressions <quantifier>` are a very powerful tool for writing
+specifications, but they can also lead to incredibly long running times.  For
+this reason, the Prover uses an approximation called "grounding".
+
+It is not possible to ground every expression perfectly.  While grounding is
+{term}`sound` (i.e. it will not allow a rule to be verified if it is not true),
+there are cases where it may generate {term}`counterexample`s even for rules
+that should pass.  For example, a counterexample may not obey a `require`
+statement that contains a quantifier.
+
+You can prevent spurious counterexamples by turning off grounding (by passing
+{ref}`-smt_groundQuantifiers`), but without grounding the Prover may run
```

Comment:
this shows up as `-settings` when it should be `-prover_args` (`-proverArgs`?), and I think nowadays we are more for conf files? not sure how this is treated in documentation.